products/neo4j.md

```diff
@@ -26,7 +26,7 @@ auto:
       fields:
         releaseCycle:
           column: "Release"
-          regex: '^(?P<value>\d+\.\d+).*$'
+          regex: '^(?P<value>\d{4}\.\d{2}\.\d)$'
```

Comment:
As far as I can tell old regex will work. New regex will capture the patch number, which is not correct for release cycles.

Reply:
Oh Ok, my bad ! will revert.